CHANGES.txt

```diff
@@ -14,6 +14,8 @@ Changes
 Fixes
 =====
 
+ - Update License Ident wording in the admin UI.
```

Comment:
I would rewrite this to be "Licence ident wording has been updated to be clearer" or something.